.github/actions/run-and-record-tests/action.yml

```diff
@@ -42,7 +42,7 @@ runs:
           --test-subdirs '${{ inputs.test-subdirs }}' \
```

Comment:
Will be good to document --test-subdirs as well and how that differs from --suite.

Reply:
@raghotham it is documented in the help but I will expand on that